packages/app/src/cli/utilities/app/http-reverse-proxy.ts

```diff
@@ -91,12 +93,20 @@ ${output.token.json(JSON.stringify(rules))}
   abortController.signal.addEventListener('abort', () => {
     server.close()
   })
+
+  async function openPreviewUrl() {
```

Comment:
Actually this doesn't work with `Ctrl+C`, I'll need to find an alternative solution.